CHANGELOG.md

```diff
@@ -1,4 +1,17 @@
 
+## Master
```

Comment:
hm something is off here because `2-2-stable` already has an empty master section https://github.com/CocoaPods/fourflusher/blob/2-2-stable/CHANGELOG.md